Review/week-3/classes-day02.md

```diff
@@ -0,0 +1,29 @@
+# Review: Classes in Ruby
+
+## Yesterday's Lessons and Activities
+* [Object-Oriented Design in Programming: Overview](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/01-ruby-fundamentals/classes-overview.md)
+* [Object Orientation: Classes in Ruby](Description:https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/01-ruby-fundamentals/classes-in-ruby.md)
+* [Defining Classes Worksheet](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/01-ruby-fundamentals/exercises/oo-worksheet.md)
+* [Helper methods and self](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/01-ruby-fundamentals/classes-part-2.md)
+* [Solar System Project](https://github.com/Ada-C12/solar-system)
+
+## Guided Practice
+
+Let's Create a class called `Animal` with the following instance variables:
+  * `type` (i.e. wolf, bear, etc.)
+  * `distance_from_home`
+
+Our class will have a couple helper methods. We'll talk together about what these methods do.
+  * `travel` 
+  * `summary`
+
+Let's build our class and instantiate a few `Animals` to practice what we've learned:
+  * the constructor
+  * helper methods
+  * attr_reader, attr_writer, attr_accessor
+  * self
```

Comment:
It' unclear how to practice `self` here.

Similar for helper methods or file organization.